docs/guidance/language.md

```diff
@@ -0,0 +1,71 @@
+# Publishing data in your own language
+
+You can publish the value of free-text fields, for example `title`, `description` and `parties/name`, in your own language. You ought to set `language` to the language used in free-text fields in your data.
+
+In order for your data to be interoperable and compatible with OC4IDS tools and methodologies, you cannot:
+
+* Translate codes from OC4IDS's codelists, for example the `type` field needs to be set to a code from the [ProjectType codelist](../reference/codelists.md#projecttype), like 'construction', which cannot be translated.
+* Translate field names (keys), for example, you cannot translate `title` to `ชื่อ`.
+
+To check whether a field can be published in your own language, check the [internationalization lookup table](#internationalization-lookup).
+
+## Examples
+
+The following JSON snippet is valid OC4IDS data. `title` can be published in Thai because it is a free-text field:
+
+```json
+{
+  "id": "1",
+  "title": "ตัวอย่างโครงการ",
+  "type": "construction"
+}
+```
+
+The following JSON snippet is not valid because "การก่อสร้าง" is not a valid code from the [ProjectType codelist](../reference/codelists.md#projecttype):
+
+```json
+{
+  "id": "1",
+  "title": "ตัวอย่างโครงการ",
+  "type": "การก่อสร้าง"
+}
+```
+
+The following JSON snippet is not valid because "ชื่อ" and "พิมพ์" are not valid field names in OC4IDS:
```

Comment:
I'm going to rework the examples into the above text, because it's fine to add fields that are not in OC4IDS (additional fields). The problem is more specifically that these are translations of the original fields, and they are substituting for those fields.

Reply:
The changes look good.